README.md

```diff
@@ -136,9 +136,9 @@ A barebones `gyp` file appropriate for building a node addon looks like:
 Some additional resources for addons and writing `gyp` files:
 
  * ["Going Native" a nodeschool.io tutorial](http://nodeschool.io/#goingnative)
- * ["Hello World" node addon example](https://github.com/joyent/node/tree/master/test/addons/hello-world)
- * [gyp user documentation](https://chromium.googlesource.com/external/gyp/+/master/docs/UserDocumentation.md)
- * [gyp input format reference](https://chromium.googlesource.com/external/gyp/+/master/docs/InputFormatReference.md)
+ * ["Hello World" node addon example](https://github.com/nodejs/node-v0.x-archive/tree/master/test/addons/hello-world)
```

Comment:
I'd link to https://github.com/nodejs/node/tree/master/test/addons/hello-world here.